.github/pull_request_template.md

```diff
@@ -26,8 +26,8 @@ your test configuration
 - Test B
 
 **Test Configuration**:
-* Operating system
-* Version number
+* Operating system: iOs
```

Comment:
Do we want to use iOs as the example?

Reply:
I chose it because it's not a real operating system, but it's clearly an operating system.  Open to other suggestions.  LCARS perhaps?

Reply:
lol.  I was largely thinking of if we were implying support for a system we don't support.  We could always go with a fictional example.

Reply:
Go ahead and pick your fave, Battlestar Galactica or Star Trek?